src/plugins/discover/public/context_awareness/profile_providers/example/example_data_source_profile/profile.tsx

```diff
@@ -74,25 +76,32 @@ export const createExampleDataSourceProfileProvider = (): DataSourceProfileProvi
         );
       },
     }),
-    getDocViewer: (prev) => (params) => {
-      const recordId = params.record.id;
-      const prevValue = prev(params);
-      return {
-        title: `Record #${recordId}`,
-        docViewsRegistry: (registry) => {
-          registry.add({
-            id: 'doc_view_example',
-            title: 'Example',
-            order: 0,
-            component: () => (
-              <div data-test-subj="exampleDataSourceProfileDocView">Example Doc View</div>
-            ),
-          });
+    getDocViewer:
+      (prev, { context }) =>
+      (params) => {
+        const recordId = params.record.id;
+        const prevValue = prev(params);
+        return {
+          title: `Record #${recordId}`,
+          docViewsRegistry: (registry) => {
+            registry.add({
+              id: 'doc_view_example',
+              title: 'Example',
+              order: 0,
+              component: () => (
+                <EuiFlyoutBody>
```

Comment:
It makes it double `EuiFlyoutBody` in the components tree but probably it's fine for the example use case.

Reply:
Oops, I didn't even notice... I can address this as a small followup to keep things cleaner and following best practices.